src/Passes/site-packages/TasksCI/settings.py

```diff
@@ -0,0 +1,17 @@
+# Copyright (c) Microsoft Corporation.
+# Licensed under the MIT License.
+
+from os import path
+import multiprocessing
+
+PROJECT_ROOT = path.abspath(path.dirname(path.dirname(path.dirname(__file__))))
+
+MAX_CONCURRENCY = 7
+
+
+def get_concurrency():
```

Comment:
`get_degree_of_concurrency` ? Not sure how python apis typically handle this

Reply:
I do not believe that Python has any standard way dealing with this. Regardless, we do want to have the ability to limit the concurrency as you may end up in the situation that your compilation process is bound by memory more than processing power. Hence a custom function to define the concurrency for the CI tasks. In time, we may expand this different functions for build, test and other tasks.